src/cui.cpp

```diff
@@ -61,17 +67,23 @@ const int MIN_COLUMN_WIDTH_DEV = 5;
 const int COLUMN_WIDTH_SENT = 11;
 const int COLUMN_WIDTH_RECEIVED = 11;
 const int COLUMN_WIDTH_UNIT = 6;
+const int  NUMBER_OF_DECIMALS_SENT = 1;
+const int  NUMBER_OF_DECIMALS_RECEIVED = 1;
```

Comment:
Why do you think showing one decimal is better than showing three? Three seems to make more sense to me.

Reply:
Hi, a few collected thoughts on decimals:
. I found nethogs a few days ago and really like it. But, the first thing that was not clear to me on the first run was "that dot is to separate thousands or decimals?". Maybe "B" stands for bit ! etc. etc.  A 3 digits block makes (at least me) immediately think of thousand separator.  The dot does not help, in central Europe this is a thousand separator.
. In general, it is of scarce interest to show even one decimal for numbers greater than, say 50. It is more noise than information since the error you introduce leaving out the decimal is very little. Still, it is noise that eats a lot of column space.
. Consider "top", there, where decimals are informative it is only 1 decimals, for magnitudes that are expected to be less than 100 most of the time.  
. Column are very scarce resource,  so if we keep long numbers (no automatic udm adjustment) the choice is between 2 decimals or 2 thousand separators. My preference is to read well the big numbers and forget about the tiny digits dance.